app/src/androidTest/java/org/simple/clinic/DatabaseMigrationAndroidTest.kt

```diff
@@ -2310,6 +2310,53 @@ class DatabaseMigrationAndroidTest {
     verifyNewStatus(uuid = uuid2, newStatus = "allowed")
     verifyNewStatus(uuid = uuid3, newStatus = "denied")
   }
+
+  @Test
+  fun migrate_userstatus_from_37_to_38() {
+    val db_v37 = helper.createDatabase(version = 37)
+
+    fun insertAppointment(uuid: String, oldStatus: String) {
+      db_v37.execSQL("""
+        INSERT OR REPLACE INTO `Appointment`
+          (`uuid`,`patientUuid`,`facilityUuid`,`scheduledDate`,`status`,`appointmentType`,`syncStatus`,`createdAt`,`updatedAt`)
+        VALUES (
```

Comment:
We don't use ` `` ` in SQL queries anymore.
Refer the official [SQLLite docs](https://www.sqlite.org/lang_keywords.html) for the legends.
`""` - for identifiers (like table and column names)
`''` - for literals (like values)

Reply:
done